app/src/test/kotlin/com/wire/android/ui/home/conversations/usecase/GetConversationsFromSearchUseCaseTest.kt

```diff
@@ -110,6 +118,13 @@ class GetConversationsFromSearchUseCaseTest {
             } returns flowOf(PagingData.from(conversations))
         }
 
-        fun arrange() = this to GetConversationsFromSearchUseCase(useCase, wireSessionImageLoader, userTypeMapper, dispatcherProvider)
+        fun arrange() = this to GetConversationsFromSearchUseCase(
+            useCase,
+            getFavoriteFolderUseCase,
+            observeConversationsFromFolderUseCase,
+            wireSessionImageLoader,
+            userTypeMapper,
+            dispatcherProvider
+        )
```

Comment:
suggestion: could you write a test to make sure that when we call this use case with `ConversationFilter.FAVORITES` then we get only favorited conversations or at least that we call `GetFavoriteFolderUseCase` and `ObserveConversationsFromFolderUseCase` instead of `GetPaginatedFlowOfConversationDetailsWithEventsBySearchQueryUseCase`? 😄